evalml/data_checks/target_leakage_data_check.py

```diff
@@ -105,7 +106,7 @@ def validate(self, X, y):
             ... ]
 
 
-            The default method can be changed to pearson from mutual information.
+            The default method can be changed to pearson from max.
```

Comment:
Isn't the default `mutual`? Should this read `The default method can be changed to pearson from mutual.`?